.github/workflows/release-go-binaries.yml

```diff
@@ -0,0 +1,54 @@
+name: Tag and Release Go Binaries
+
+on:
+  push:
+    # paths: ["version/.ts-bridge-version"]
```

Comment:
is this a comment you're meant to remove?

Reply:
Oops yes thanks for picking that up